### PR TITLE
[Editor] fix widget's inspector bug

### PR DESCRIPTION
**Pull request by chichinohaha**
1 use vue.min.js instead of  vue.js.
    - remove vue's tips.
2、 use an invisible <ui-prop> element to dispatch "change-dump"

Re: cocos-creator/3d-tasks#

### CODE REVIEW

**Review by arsen2010 — APPROVED** [review]

File: editor/inspector/components/widget.js [hunk 1]
```diff
@@ -549,11 +549,8 @@ exports.methods = {
         this.dump.values && this.dump.values.forEach((dump) => {
             dump.value[key].value = value;
         });
-        Editor.Message.send('scene', 'set-property', {
-            uuid: this.dump.value.uuid.value,
-            path: key,
-            dump: this.dump.value[key],
-        });
+        this.$refs['summitProp'].dump = this.dump.value[key];
+        this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]:
```typescript
xxx;
```

File: editor/inspector/components/widget.js [hunk 2]
```diff
@@ -607,11 +604,8 @@ exports.methods = {
             }
         }
         const { path, dump } = update(this.dump, true);
-        Editor.Message.send('scene', 'set-property', {
-            uuid: this.dump.value.uuid.value,
-            path,
-            dump,
-        });
+        this.$refs['summitProp'].dump = dump.value[path];
+        this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 3]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
                 if (dump.value.isAlignLeft.value !== horizontal.isAlignLeft.value) {
                     dump.value.isAlignLeft.value = horizontal.isAlignLeft.value;
-
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignLeft',
-                        dump: this.dump.value.isAlignLeft,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignLeft;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 4]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
                 }
                 if (dump.value.isAlignRight.value !== horizontal.isAlignRight.value) {
                     dump.value.isAlignRight.value = horizontal.isAlignRight.value;
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignRight',
-                        dump: this.dump.value.isAlignRight,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignRight;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 5]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
                 }
                 if (dump.value.isAlignHorizontalCenter.value !== horizontal.isAlignHorizontalCenter.value) {
                     dump.value.isAlignHorizontalCenter.value = horizontal.isAlignHorizontalCenter.value;
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignHorizontalCenter',
-                        dump: this.dump.value.isAlignHorizontalCenter,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignHorizontalCenter;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 6]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
             if (vertical) {
                 if (dump.value.isAlignTop.value !== vertical.isAlignTop.value) {
                     dump.value.isAlignTop.value = vertical.isAlignTop.value;
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignTop',
-                        dump: this.dump.value.isAlignTop,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignTop;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 7]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
                 }
                 if (dump.value.isAlignVerticalCenter.value !== vertical.isAlignVerticalCenter.value) {
                     dump.value.isAlignVerticalCenter.value = vertical.isAlignVerticalCenter.value;
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignVerticalCenter',
-                        dump: this.dump.value.isAlignVerticalCenter,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignVerticalCenter;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 8]
```diff
@@ -767,56 +761,39 @@ exports.methods = {
...
                 }
                 if (dump.value.isAlignBottom.value !== vertical.isAlignBottom.value) {
                     dump.value.isAlignBottom.value = vertical.isAlignBottom.value;
-                    Editor.Message.send('scene', 'set-property', {
-                        uuid: this.dump.value.uuid.value,
-                        path: 'isAlignBottom',
-                        dump: this.dump.value.isAlignBottom,
-                    });
+                    this.$refs['summitProp'].dump = dump.value.isAlignBottom;
+                    this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

File: editor/inspector/components/widget.js [hunk 9]
```diff
@@ -852,12 +829,9 @@ exports.methods = {
...
         this.dump.values && this.dump.values.forEach((dump) => {
             dump.value._lockFlags.value = lockValue;
-
-            Editor.Message.send('scene', 'set-property', {
-                uuid: this.dump.value.uuid.value,
-                path: '_lockFlags',
-                dump: this.dump.value._lockFlags,
-            });
+            this.$refs['summitProp'].dump = dump.value._lockFlags;
+            this.$refs['summitProp'].dispatch('change-dump')
```
Comment (VisualSJ) [inline editor/inspector/components/widget.js]: ;

**Comment by VisualSJ** [comment]
Please change the title to "[Editor] Fix widget inspector bug"

**Review by VisualSJ — APPROVED** [review]

File: editor/inspector/components/widget.js (hunk 9 above)
Comment (chichinohaha) [reply editor/inspector/components/widget.js]: fixed

**Pull request merged**